src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -305,10 +350,19 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
           // Then mask it if appropriate
           val (base, qual) = if (aBase == NoCall || bBase == NoCall || rawQual == PhredScore.MinValue) (NoCall, NoCallQual) else (rawBase, rawQual)
 
-          bases(i)  = base
-          quals(i)  = qual
-
-          errors(i) = min(sourceReads.count(s => s.length > i && isError(s.bases(i), rawBase)), Short.MaxValue).toShort
+          bases(i) = base
+          quals(i) = qual
+
+          // NB: optimized based on profiling; was previously:
```

Comment:
This makes me sad.  Is it really faster-enough to matter?

Reply:
Yup.